pom.xml

```
@@ -25,7 +25,7 @@
         <prosessering.version>1.20230207133526_5430e48</prosessering.version>
         <felles.version>1.20230116145510_2afcc20</felles.version>
         <eksterne-kontrakter-bisys.version>2.0_20221118130052_e2abd9f</eksterne-kontrakter-bisys.version>
-        <felles-kontrakter.version>2.0_20230223121809_a777026</felles-kontrakter.version>
+        <felles-kontrakter.version>3.0_20230314075921_27fa962-JAKARTA</felles-kontrakter.version>
```

Hørte med Johan ang. jakarta vs javax i familie-tilbake, og der endret vi den til siste javax-versjon for å være på den trygge siden
```suggestion
        <felles-kontrakter.version>2.0_20230313155154_f9438bd</felles-kontrakter.version>
```

---

Jakarta er spring boot 3